src/main/app/src/assets/i18n/nl.json

```diff
@@ -539,6 +539,8 @@
   "msg.error.invalid.bsn": "Het {{label}}-veld bevat een ongeldig burgerservicenummer. Vul een valide burgerservicenummer in",
   "msg.error.invalid.vestigingsnummer": "Het {{label}}-veld bevat een ongeldig vestigingsnummer. Vul een valide vestigingsnummer in",
   "msg.error.invalid.kvk": "Het {{label}}-veld bevat een ongeldig KvK-nummer. Vul een valide KvK-nummer in",
+  "msg.error.invalid.huisnummer": "Het {{label}}-veld mag alleen bestaan uit cijfers. Eventuele toevoegingen worden ook gevonden.",
```

Comment:
Maak de volgorde van de messages hier gelijk aan die in en.json. Wat de juiste volgorde is maakt niet uit als het maar gelijk is.

Reply:
De gehele volgorde van de msg.error is een zooitje. Mijne stond goed, maar heb de rest even verplaatst en er nog 1 bij gezet die mist.